docs/en/compute/Hadoop-MapReduce.md

```diff
@@ -158,11 +162,11 @@ Hadoop MapReduce users can add `"-Dproperty=value"` after the `hadoop jar` or `y
 and the properties will be propagated to all the tasks of this job.  For example, the following
 MapReduce job of wordcount sets write type to `CACHE_THROUGH` when writing to Alluxio:
 
-```bash
-./bin/hadoop jar libexec/share/hadoop/mapreduce/hadoop-mapreduce-examples-2.7.3.jar wordcount \
--Dalluxio.user.file.writetype.default=CACHE_THROUGH \
--libjars {{site.ALLUXIO_CLIENT_JAR_PATH}} \
-<INPUT FILES> <OUTPUT DIRECTORY>
+```console
+$ ./bin/hadoop jar libexec/share/hadoop/mapreduce/hadoop-mapreduce-examples-2.7.3.jar wordcount \
+  -Dalluxio.user.file.writetype.default=CACHE_THROUGH \
+  -libjars {{site.ALLUXIO_CLIENT_JAR_PATH}} \
+  <INPUT FILES> <OUTPUT DIRECTORY>
```

Comment:
Should we make indenting newlines in these console blocks standard? Most of the time we don't do this, but I like when they are indented

Reply:
agree. better for readability. I will change then when I see these